src/main/resources/hudson/plugins/git/extensions/impl/DisableAutomaticScheduling/help.html

```diff
@@ -0,0 +1,6 @@
+<div>
+       Git plugin triggers another build when multiple new revisions are detected.
+       This can lead to many unwanted builds.<br><br>
+       If this option is selected, this trigger will be disabled and only one revision will get build. <br>
+       This is very helpful if you are triggering the builds only by an external trigger via notifyCommit url.
```

Comment:
Maybe elaborate this further?

It might be helpful to document possible use cases, like the one I mentioned in my PR comment.